pkg/minikube/cluster/cluster_test.go

```diff
@@ -67,15 +68,17 @@ func TestCreateHost(t *testing.T) {
 	}
 
 	found := false
-	for _, driver := range constants.SupportedVMDrivers {
-		if h.DriverName == driver {
+	drivers := []string{}
```

Comment:
If you add a "String()" method to DriverDef that returns the .Name field, I think you can simplify this a bit and avoid collecting the drivers slice.